Jenkinsfile

```diff
@@ -0,0 +1,41 @@
+def fail(reason) {
+  def pr_branch = ''
```

Comment:
Looks like this line and the next three are unneeded; we could probably do without this function.

Reply:
function removed